sdk/openai/openai/customts/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "OpenAI",
+  "version": "0.1.0",
+  "type": "module",
+  "dependencies": {
+    "@azure-tools/cadl-apiview": "latest",
```

Comment:
I believe all these packages should be pinned to exact versions published on npmjs.com.  Not "latest" or redirect links.